kbase-extension/static/kbase/js/common/errorDisplay.js

```diff
@@ -77,6 +115,34 @@ define(['bluebird', 'uuid', 'common/html'], (Promise, Uuid, html) => {
                   )
                 : '',
 
+            errorObject.errorDump
+                ? div(
+                      {
+                          class: `${cssBaseClass}__error_dump_container`,
+                      },
+                      [
+                          div(
+                              {
+                                  class: `${cssBaseClass}__error_dump_title collapsed`,
+                                  role: 'button',
+                                  dataToggle: 'collapse',
+                                  dataTarget: `#${uniqueID}__dump`,
+                                  ariaExpanded: 'false',
+                                  ariaControls: `${uniqueID}__dump`,
+                              },
+                              [span({}, ['Raw error JSON'])]
+                          ),
+                          pre(
+                              {
+                                  class: `${cssBaseClass}__error_dump_code collapse`,
+                                  id: `${uniqueID}__dump`,
+                              },
+                              [JSON.stringify(errorObject.errorDump, null, 1)]
```

Comment:
Instead of dumping the error to the console, where it would probably languish unseen for many decades, I'm dumping it as JSON in the UI. At least there will be something that users can easily copy and paste into bug reports.